ui/components/multichain/create-snap-account/create-snap-account.tsx

```diff
@@ -45,15 +45,15 @@ export const CreateSnapAccount = ({
   const snapClient = useMultichainWalletSnapClient(clientType);
 
   const onCreateAccount = useCallback(
-    async (suggestedName?: string) => {
+    async (_accountNameSuggestion?: string) => {
```

Comment:
Use of a `_` prefix, to avoid linter issue if `multi-srp` feature flag isn't defined.